.dockerignore

```diff
@@ -0,0 +1,13 @@
+.dockerignore
+*.pyc
```

Comment:
@jkinkead none of these seem to exclude `pyc` files from my docker builds.  The other ignores work fine.